src/api/route-services/pipelines.js

```diff
@@ -0,0 +1,196 @@
+const AWS = require('aws-sdk');
+const crypto = require('crypto');
+const fetch = require('node-fetch');
+const YAML = require('yaml');
+const jq = require('jq-web');
+const config = require('../../config');
+
+
+class PipelinesService {
+  constructor() {
+    this.stepFunctions = new AWS.StepFunctions({
+      region: config.awsRegion,
+    });
+
+    this.clusterInfo = null;
+  }
+
+  async getClusterInformation() {
+    if (!this.clusterInfo) {
+      const eks = new AWS.EKS({
+        region: config.awsRegion,
+      });
+
+      const accountId = await config.awsAccountIdPromise();
+      const info = await eks.describeCluster({ name: `biomage-${config.clusterEnv}` }).promise();
```

Comment:
we need some error handling here

Reply:
Same question as before. If the cluster cannot be found, an exception will be thrown and there is no way to recover from it.

Reply:
yup, as discussed in the other thread, sounds good